src/data/fintype.lean

```diff
@@ -637,11 +637,8 @@ def perms_of_list : list α → list (perm α)
 
 lemma length_perms_of_list : ∀ l : list α, length (perms_of_list l) = l.length.fact
 | []       := rfl
-| (a :: l) := begin
-  rw [length_cons, nat.fact_succ],
-  simp [perms_of_list, length_bind, length_perms_of_list, function.comp, nat.succ_mul],
-  cc
-end
+| (a :: l) := by rw [length_cons, nat.fact_succ];
```

Comment:
Such multiline `;`-chains of tactics are discouraged. If it fits on 1 line, then `by { foo, bar }` is fine, otherwise a `begin`-`end` block is preferred.

Reply:
I have no idea why this change shows up on this PR.  This PR doesn't change `fintype.lean` at all.  I'll change it to `begin .. end` anyhow.